### ISSUE TITLE
gamepad/joystick support

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1308]
I have no idea what gamepads and joysticks look like from an input perspective (i doubt it's all `STICK` and `STRIG`, heh, a little nostalgia for my fellow ATARI programmers there...), but we ought support them.

**Comment by WSLUser** [comment]
AtariOS should provide a nice reason to support (It is an Ubuntu derivative). There are indeed fellow Atari game programmers who are writing for the new Atari VCS. Supporting notcurses there through use of the controller/joystick would be epicly awesome.

**Comment by dankamongmen** [comment]
i'd very much like to avoid using SDL. let me rephrase that: we're definitely not taking a dep on SDL for joystick support,

**Comment by dankamongmen** [comment]
```
[schwarzgerat](0) $ ldd `which jstest`
	linux-vdso.so.1 (0x00007fff75f82000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f57cead3000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f57cece9000)
[schwarzgerat](0) $
```

that's promising
